core-rust/state-manager/src/store/db.rs

```diff
@@ -130,7 +132,8 @@ impl StateManagerDatabase {
             debug!("Running genesis on the engine...");
             let mut db_txn = state_manager_db.create_db_transaction();
 
-            let genesis_receipt = bootstrap(&mut db_txn).expect("Genesis wasn't run");
+            let interpreter = ScryptoInterpreter::<DefaultWasmEngine>::default();
```

Comment:
This uses a different interepreter (and eg different metering config) compared with the state manager.

Now I'm not sure _why_ we overwrite the metering config - this should probably be changed to just be the default? In case genesis ends up deploying / running packages - which it might do to be fair?

We should probably create the ScryptoInterpreter in state manager init and then pass the same one into DB and State Manager set up?

Reply:
This is cleaned up in the next PR which focuses on cleaning up genesis